docs/tools/index.rst

```diff
@@ -1,12 +1,13 @@
-Tools for Aiven
-===============
+Aiven Tools
```

Comment:
This is fine for the beta, however I would add some "squares" like the home page to make options more visible

Reply:
One square for each tool

Reply:
I like this idea but I'm reluctant to delay this PR for it. Let's add it separately!